ttnn/ttnn/device.py

```diff
@@ -138,11 +138,47 @@ def is_grayskull(device):
     return device.arch() == ttnn._ttnn.device.Arch.GRAYSKULL
 
 
+def is_blackhole(device):
+    return device.arch() == ttnn._ttnn.device.Arch.BLACKHOLE
+
+
 SetDefaultDevice = ttnn._ttnn.device.SetDefaultDevice
 GetDefaultDevice = ttnn._ttnn.device.GetDefaultDevice
 format_input_tensor = ttnn._ttnn.device.format_input_tensor
 format_output_tensor = ttnn._ttnn.device.format_output_tensor
 pad_to_tile_shape = ttnn._ttnn.device.pad_to_tile_shape
 
 
+def GetComputeKernelConfig(
```

Comment:
This function does not get a config, it creates it. Should it be really called "get"?